docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -1,86 +1,94 @@
-`2.7 Release Notes`_                    Unmaintained (end of life)
-`2.6 Release Notes`_                    Unmaintained (end of life)
-`2.5 Release Notes`_                    Unmaintained (end of life)
-<2.5                                    Unmaintained (end of life)
+`3.0 Release Notes`_                    Current
+`2.10 Release Notes`_                   Unmaintained (end of life)
 ==================================      =================================================
 
 
-.. Comment: devel used to point here but we're currently revamping our changelog process and have no
-   link to a static changelog for devel _2.6: https://github.com/ansible/ansible/blob/devel/CHANGELOG.md
+.. _3.0 Release Notes:
 .. _2.10 Release Notes:
 .. _2.10: https://github.com/ansible-community/ansible-build-data/blob/main/2.10/CHANGELOG-v2.10.rst
-.. _2.9 Release Notes:
-.. _2.9: https://github.com/ansible/ansible/blob/stable-2.9/changelogs/CHANGELOG-v2.9.rst
-.. _2.8 Release Notes:
-.. _2.8: https://github.com/ansible/ansible/blob/stable-2.8/changelogs/CHANGELOG-v2.8.rst
-.. _2.7 Release Notes: https://github.com/ansible/ansible/blob/stable-2.7/changelogs/CHANGELOG-v2.7.rst
-.. _2.6 Release Notes:
-.. _2.6: https://github.com/ansible/ansible/blob/stable-2.6/changelogs/CHANGELOG-v2.6.rst
-.. _2.5 Release Notes: https://github.com/ansible/ansible/blob/stable-2.5/changelogs/CHANGELOG-v2.5.rst
 
 
-ansible-core release cycle
---------------------------
+Release cycle for ansible-core
+-------------------------------
 
-``ansible-core`` is developed and released on a flexible release cycle.
-This cycle can be extended in order to allow for larger changes to be properly
-implemented and tested before a new release is made available. See :ref:`roadmaps` for upcoming release details.
+``ansible-core`` is developed and released on a flexible release cycle. This cycle can be extended in order to allow for larger changes to be properly implemented and tested before a new release is made available. See :ref:`roadmaps` for upcoming release details.
 
 ``ansible-core`` has a graduated maintenance structure that extends to three major releases.
 For more information, read about the :ref:`development_and_stable_version_maintenance_workflow` or
 see the chart in :ref:`release_schedule` for the degrees to which current releases are maintained.
 
-If you are using a release of ``ansible-core`` that is no longer maintained, we strongly
-encourage you to upgrade as soon as possible in order to benefit from the
-latest features and security fixes.
-
-Older, unmaintained versions of ``ansible-core`` can contain unfixed security
-vulnerabilities (*CVE*).
+Older, unmaintained versions of ``ansible-core`` can contain unfixed security vulnerabilities (*CVEs*). If you are using a release of ``ansible-core`` that is no longer maintained, we strongly encourage you to upgrade as soon as possible to benefit from the latest features and security fixes.
```

Comment:
same suggestion on  turning this into a note to highlight its importance.

Reply:
Done.